plugins/utils/repeat.nix

```
@@ -0,0 +1,17 @@
+{
+  lib,
+  pkgs,
+  ...
+}:
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "repeat";
+  originalName = "vim-repeat";
+  defaultPackage = pkgs.vimPlugins.vim-repeat;
```

Now that #2139 is merged:
```suggestion
  package = "vim-repeat";
```